app/src/main/java/org/p2p/wallet/jupiter/repository/model/JupiterSwapToken.kt

```diff
@@ -11,4 +12,10 @@ data class JupiterSwapToken(
     val tokenName: String,
     val tokenSymbol: String,
     val tags: List<String>
-)
+) {
+    fun isSol(): Boolean = tokenMint.base58Value == Constants.WRAPPED_SOL_MINT
```

Comment:
I created SolAddress wrapper class, I think we can try to cast it there, and then call isSol fun